model/CMakeLists.txt

```
@@ -409,12 +409,33 @@ foreach (xlen IN ITEMS 32 64)
                         --coq-lib mem_metadata
                         --coq-output-dir ${CMAKE_CURRENT_BINARY_DIR}
                         # The prefix of the output files.
-                        -o "riscv_${arch}"
+                        -o "coq_${arch}"
                         # Input files.
                         ${sail_srcs}
                 )
                 add_custom_target(generated_coq_${arch} DEPENDS "coq_${arch}.?")
             endif()
+
+            # Build Lean definitions
+            if (variant STREQUAL "lean")
+                string(TOUPPER ${arch} arch_uppercase) 
+                add_custom_command(
```

```suggestion
                string(TOUPPER ${arch} arch_uppercase)
                add_custom_command(
```

Fix trailing whitespace.